apps/web/src/pages/templates/components/TemplateEditorFormProvider.tsx

```diff
@@ -192,10 +192,8 @@ const TemplateEditorFormProvider = ({ children }) => {
           },
         });
         setTrigger(response.triggers[0]);
-        reset({
-          ...form,
-          steps: response.steps,
-        });
+        reset(mapNotificationTemplateToForm(response));
```

Comment:
@BiswaViraj could you please make sure the bug you described [here](https://github.com/novuhq/novu/pull/3524/files#r1208925796) is still resolved? 
I was not able to reproduce what you described

Reply:
I've also checked and wasn't able to reproduce it.